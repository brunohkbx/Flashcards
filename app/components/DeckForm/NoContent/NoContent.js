import React from 'react';
import { Image } from 'react-native';
import { Colors } from 'react-native-paper';
import CenteredContainer from '../../../components/CenteredContainer';
import Avatar from '../../../components/Avatar';
import AvatarDescription from '../../../components/AvatarDescription';

const NoContent = () => {
  return (
    <CenteredContainer>
      <Avatar
        width={100}
        height={100}
        borderRadius={50}
        backgroundColor={Colors.white}
        style={{ marginBottom: 15 }}
      >
        <Image
          source={require('../../../assets/images/flashcard.png')}
          style={{ width: 64, height: 64 }}
        />
      </Avatar>
      <AvatarDescription>No flashcards</AvatarDescription>
    </CenteredContainer>
  );
};

export default NoContent;
