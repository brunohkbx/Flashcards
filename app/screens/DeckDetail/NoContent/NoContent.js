import React from 'react';
import { Caption, Colors } from 'react-native-paper';
import CenteredContainer from '../../../components/CenteredContainer';
import Avatar from '../../../components/Avatar';
import { FlashcardIcon } from '../../../components/Icons';

const NoContent = () => {
  return (
    <CenteredContainer>
      <Avatar
        width={100}
        height={100}
        borderRadius={50}
        backgroundColor={Colors.white}
      >
        <FlashcardIcon size={50} />
      </Avatar>
      <Caption style={{ fontSize: 18}}>No flashcards</Caption>
    </CenteredContainer>
  );
};

export default NoContent;
