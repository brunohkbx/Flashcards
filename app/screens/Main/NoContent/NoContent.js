import React from 'react';
import { Image } from 'react-native';
import { Caption, Colors } from 'react-native-paper';
import CenteredContainer from '../../../components/CenteredContainer';
import Avatar from '../../../components/Avatar';

const NoContent = () => {
  return (
    <CenteredContainer>
      <Avatar
        width={100}
        height={100}
        borderRadius={50}
        backgroundColor={Colors.white}
      >
        <Image
          source={require('../../../assets/images/card_stack.png')}
          style={{ width: 64, height: 64 }}
        />
      </Avatar>
      <Caption style={{ fontSize: 18}}>No decks</Caption>
    </CenteredContainer>
  );
};

export default NoContent;
