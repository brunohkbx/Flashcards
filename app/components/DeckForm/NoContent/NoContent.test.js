import React from 'react';
import { shallow } from 'enzyme';
import NoContent from './NoContent';

describe('NoContent', () => {
  it('renders properly', () => {
    const wrapper = shallow(<NoContent />);

    expect(wrapper).toMatchSnapshot();
  });
});
