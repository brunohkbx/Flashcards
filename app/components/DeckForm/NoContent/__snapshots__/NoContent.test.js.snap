// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContent renders properly 1`] = `
<Styled(Component)>
  <Styled(Component)
    backgroundColor="#ffffff"
    borderRadius={50}
    height={100}
    width={100}
  >
    <Image
      source={1}
      style={
        Object {
          "height": 64,
          "width": 64,
        }
      }
    />
  </Styled(Component)>
  <Caption
    style={
      Object {
        "fontSize": 18,
      }
    }
  >
    No flashcards
  </Caption>
</Styled(Component)>
`;
